aiida/cmdline/params/options/__init__.py

```diff
@@ -451,7 +452,43 @@ def decorator(command):
 
 FREQUENCY = OverridableOption('-F', '--frequency', 'frequency', type=click.INT)
 
-VERBOSE = OverridableOption('-v', '--verbose', is_flag=True, default=False, help='Be more verbose in printing output.')
+VERBOSE = OverridableOption('-v', '--verbose', count=True, help='Print more verbose output.')
+
+
+def VERBOSITY(logger='cli', *names, **kwargs):  # pylint: disable=keyword-arg-before-vararg,invalid-name
+    '''A decorator that adds a `--verbosity, -v` option to the decorated
+    command.
+
+    Name can be configured through ``*names``. Keyword arguments are passed to
+    the underlying ``click.option`` decorator.
+    '''
+    from aiida.common.log import LOG_LEVELS
+    levels_str = ', '.join(LOG_LEVELS.keys())
+
+    if not names:
+        names = ['--verbosity', '-v']
+
+    kwargs.setdefault('default', 'INFO')
```

Comment:
Do we maybe want to line this up with the `logging.aiida_loglevel` config option? Actually, I notice you defined a new logging handler `click`. Should we maybe create a new `logging.verdi_loglevel` option that controls this default?

Reply:
> Do we maybe want to line this up with the logging.aiida_loglevel config option? 

The AiiDA logleve defaults to the custom `REPORT` value, which is in between `INFO` and `WARNING`.
For the cli, however, we *do* want to see what `echo.echo_info()` prints by default, i.e. I think we should default to a log level of `INFO` or lower.

> Should we maybe create a new logging.verdi_loglevel option that controls this default?

That sounds like a good idea.